docs/components/base/_index.md

```diff
@@ -12,201 +13,35 @@ imageAlt: "base"
 
 A *base* is the platform that the other parts of a mobile robot attach to.
 
-Configure your robot's *base* component to reference any *motor* components attached to the base.
 By configuring a base component, organizing individual components to produce coordinated movement, you gain an interface to control the movement of the whole physical base of the robot without needing to send separate commands to individual motors.
 
-<img src="../img/base/base-trk-rover-w-arm.png" alt="A robot comprised of a base (motors, wheels and chassis) as well as some other components. The wheels are highlighted to indicate that they are part of the concept of a 'base', while the non-base components are not highlighted. The width and circumference are required attributes when configuring a base component." />
+<img src="img/base-trk-rover-w-arm.png" alt="A robot comprised of a wheeled base (motors, wheels and chassis) as well as some other components. The wheels are highlighted to indicate that they are part of the concept of a 'base', while the non-base components are not highlighted. The width and circumference are required attributes when configuring a base component."/>
 
 Most mobile robots with a base need at least the following hardware:
 
-- A [board component](/components/board/) that can run a `viam-server` instance.
-   For example, a Raspberry Pi, or another model of single-board computer with GPIO (general purpose input/output) pins.
-
+- A [board](/components/board/).
 - Some sort of actuators to move the base.
```

Comment:
or legs?  Or treads (but those are kind of like wheels)

Reply:
Those include/are actuators though? So I think this encompasses them all.